bin/pycbc_convertinjfiletohdf

```diff
@@ -179,14 +180,11 @@ else:
         injclass = LVKNewStyleInjectionSet(args.injection_file)
     inj_file.close()
 
-if injclass is None:
-    # Assume this is a PyCBC format
-    injclass = InjectionSet(args.injection_file)
-    data = {}
-    for key in xinj.table[0].__slots__:
-        data[str(key)] = numpy.array([getattr(t, key) for t in xinj.table])
-else:
+if injclass is not None:
+    # Assume the injection is a XML file or HDF file (LVK format)
```

Comment:
No assumption here, so you can remove the "Assume"

```suggestion
    # The injection file is of known type
```

Reply:
@GarethCabournDavies Fixed as suggested. Thanks